docs/log_collection.md

```diff
@@ -46,22 +46,39 @@ github "DataDog/dd-sdk-ios"
 
 2. Initialize the library with your application context and your [Datadog client token][2]. For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-ios` library as they would be exposed client-side in the iOS application IPA byte code. For more information about setting up a client token, see the [client token documentation][2]:
 
-    {{< tabs >}}
-    {{% tab "US" %}}
-
+{{< site-region region="us" >}}
+{{< tabs >}}
+{{% tab "Swift" %}}
 ```swift
 Datadog.initialize(
     appContext: .init(),
     trackingConsent: trackingConsent,
     configuration: Datadog.Configuration
         .builderUsing(clientToken: "<client_token>", environment: "<environment_name>")
         .set(serviceName: "app-name")
+        .set(endpoint: .us1)
         .build()
 )
 ```
+{{% /tab %}}
+{{% tab "Objective-C" %}}
+```objective-c
+DDConfigurationBuilder *builder = [DDConfiguration builderWithClientToken:@"<client_token>"
+                                                                  environment:@"<environment_name>"];
```

Comment:
`:`s are not aligned, let's align them.
maybe we can do it in one single line to avoid formatting issues:
```suggestion
DDConfigurationBuilder *builder = [DDConfiguration builderWithClientToken:@"<client_token>" environment:@"<environment_name>"];
```